node/sc/subbridge.go

```diff
@@ -521,8 +527,28 @@ func (pm *SubBridge) handle(p BridgePeer) error {
 	}
 }
 
+func (sc *SubBridge) resetBridgeLoop() {
+	ticker := time.NewTicker(resetBridgeCycle)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-sc.quitSync:
+			return
+		case <-ticker.C:
+			if atomic.CompareAndSwapInt64(&sc.checkConnection, 1, 0) {
+				if sc.peers.Len() < 1 {
+					atomic.StoreInt64(&sc.checkConnection, 1)
+					continue
+				}
+				sc.bridgeManager.ResetAllSubscribedEvents()
```

Comment:
After this, if it is succeeded, how about set the status of ResetAllSubscribedEvents?

Reply:
As we discussed, connection and subscription are orthogonal events.